examples/proc_fieldSweep.py

```diff
@@ -1,142 +1,75 @@
-    # the scans to determine the relative offsets, which would give both the
-    # relative offsets and the frequency shifts needed to align the signals, at
-    # the end.
-    all_offsets = zeros(len(s.getaxis("indirect")))
-    for z in range(len(s.getaxis("indirect"))):
-        fl.plot(s["indirect", z], ax=ax_list[1])
-        if z == 0:
-            ax_list[1].axvline(color="k", x=freq_slice[0])
-            ax_list[1].axvline(color="k", x=freq_slice[-1])
-        offset = s["indirect", z].C.mean("nScans").argmax("t2").item()
-        all_offsets[z] = offset
-        carrier_freq_MHz = s.getaxis("indirect")[z]["carrierFreq"]
-        field = s.getaxis("indirect")[z]["Field"]
-        nu_rf = carrier_freq_MHz - offset / 1e6  # because in order to make my
-        #                                       offset more negative, I
-        #                                       increase my field, or decrease
-        #                                       my carrier -- in other words
-        #                                       the frequency axis for our FID
-        #                                       is the negative of (ν_RF-ν_carrier)
-        nu_NMR.append(nu_rf)
-    ax_list[1].set_title("Field Slicing")
-    s.ift("t2")
-    s *= exp(
-        -1j
-        * 2
-        * pi
-        * nddata(all_offsets, [-1], ["indirect"])
-        * s.fromaxis("t2")
+    nu_B12 = s.get_prop("acq_params")["uw_dip_center_GHz"]
+    use_freq = True
+    if use_freq:
+        s["indirect"] = s["indirect"]["carrierFreq"]
+        s.set_units("indirect", "MHz")
+        s["indirect"] = s["indirect"] / nu_B12
+    else:
+        # I wanted to use the carrier (use_frep=True), but it seems like the
+        # first point isn't stored properly
+        s["indirect"] = s["indirect"]["Field"]
+        s.set_units("indirect", "G")
+    s, ax4 = prscr.rough_table_of_integrals(s, fl=fl)
+    ax4.text(
+        0.5,
+        0.5,
+        "Warning!!!, I'm using uw_dip_center_GHz as the microwave\n"
+        "frequency, but there is no guarantee that it is!!!  It is only the\n"
+        "input to dip_lock!!\n",
+        alpha=0.5,
+        color="r",
+        va="center",
+        ha="center",
+        transform=ax4.transAxes,
     )
-    s.ft("t2")
-    # }}}
-    fl.plot(s.C.mean("nScans"), ax=ax_list[2])
-    s = s["t2":freq_slice].mean("nScans").integrate("t2")
-    # }}}
-    # {{{convert x axis to ppt = v_NMR/v_ESR
-    ppt = nu_NMR / nu_B12_GHz
-    s.setaxis("indirect", ppt)
-    s.set_units("indirect", "MHz/GHz")
-    s.rename("indirect", "resonance ratio")
-    # }}}
-    # {{{Fitting
-    fl.plot(s, "o-", ax=ax_list[3])
-    ax_list[3].set_title("Field Sweep ppt")
-    fitting = s.polyfit("resonance ratio", order=2)
-    field_fine = nddata(
-        r_[
-            s.getaxis("resonance ratio")[0] : s.getaxis("resonance ratio")[
-                -1
-            ] : 100j
-        ],
-        "resonance ratio",
+    c_poly = s.polyfit("indirect", 4)
```

Comment:
I was having trouble with this and earlier versions of this line but resolved this (needed to pull recent changes in pyspecdata)